### ISSUE TITLE
"Parse" function missing after build: `this.$papa.parse is not a function` 

### ISSUE THREAD

**Issue by marguerrrite** [twickstrom/vue-papa-parse#7]
**Overview**
I am putting together a site utilizing Vue 3, Vite, and reading a local CSV with vue-papa-parse:
* Repo: https://github.com/margueriteroth/vue-d3-dataviz-starter
* Netlify build (with error in console): https://vue-d3-dataviz-starter.netlify.app/

---
The site reads and loads CSV data when running locally. 

After running `npm build` and viewing the built site, the console is throwing this `this.$papa.parse is not a function` error. 
![image](https://user-images.githubusercontent.com/5288685/174128441-759bb6e9-4a1a-459b-98c5-1364a6b11154.png)
I am able to log the function itself but it is only showing me `dedupe` and `download`. I'm wondering if I am missing a dependency somewhere. 

Chart component utilizing `this.$papa.parse`: [MassShootingPlot.vue](https://github.com/margueriteroth/vue-d3-dataviz-starter/blob/main/src/components/MassShootingPlot/styled/MassShootingPlot.vue)

Registration of library [main.js](https://github.com/margueriteroth/vue-d3-dataviz-starter/blob/main/src/main.js/#L44).

---

**To Reproduce**
1. Pull repo and install (instructions in [ReadMe](https://github.com/margueriteroth/vue-d3-dataviz-starter/blob/main/src/main.js/#L44))
2. To view locally, run `npm run dev`,  navigate to http://localhost:3000/ to view charts
3. Build the site with `npm run build`
4. View built site `npm run preview`, navigate to http://localhost:8888/
5. See error above in browser console and blank chart

---

**Expected behavior**
`this.$papa.parse...` to load CSV data successfully, like on localhost:
![image](https://user-images.githubusercontent.com/5288685/174129497-f234aa99-2178-49df-a575-763d9dd58422.png)

I'm not sure why the `.parse` function is missing.

---

**Desktop (please complete the following information):**
 - OS: MacOs Monterrey 12.2.1
 - Chrome 102.0.5005.115 (Official Build) (x86_64)

**Comment by twickstrom** [comment]
I am able to reproduce this issue on multiple projects. I am resolving now.

**Comment by twickstrom** [comment]
@margueriteroth thank you for reporting this issue.

I have now resolved the issue with `vue-papa-parse` version `3.1.0`

I have tested this on your project as well.

I also have a couple of recommendations:
First for your .csv files. I would move those static assets to the public folder by moving the data folder from `src` to `public` and then update the path to all referenced URLs.

That way when changing: `localDataUrl: "src/data/noaa-data.csv",` to `localDataUrl: "data/noaa-data.csv",` will always resolve when running in both dev and prod.

Love your project, keep up the hard work and keep fighting the good fight!

PS Sorry this project caused you a snag.

-TW
